Source/iop/Iop_SubSystem.cpp

```diff
@@ -258,9 +256,13 @@ uint32 CSubSystem::WriteIoRegister(uint32 address, uint32 value)
 	{
 		return m_spu2.WriteRegister(address, value);
 	}
+	else if((address >= 0x1f801000 && address <= 0x1f801020) || (address >= 0x1f801400 && address <= 0x1f801420))

```

Comment:
Please use capital letters for hexadecimal numbers.

Reply:
Done